### ISSUE TITLE
Populator returns wrong input

### ISSUE THREAD

**Issue by dolfje** [formers/former#314]
When populating the former object and asking for the values, the populator gives back wrong results, if the key doesn't exist in the values.

```
Former::populate(array(
  "name" => "users",
  "fields" => array(
    0 => array(
      "name" => "test1",
      "type" => "text",
    ),
    1 => array(
      "name" => "test2",
      "type" => "text",
    ),
  ),
));

var_dump(Former::getValue("fields[2]"));

// returns: array(2) { [0]=> NULL [1]=> NULL } 
```

**Comment by anedisi** [comment]
this is still happening. 
i get a input value of ", , " because it does implode on array values, i did a quick fix where i filter the null. it is weird that nobody noticed this.

**Comment by claar** [comment]
@dolfje @anedisi Can you post both your `populate()` code as well as the code for the fields you're trying to populate?

**Comment by claar** [comment]
Fixed in #493.
